test/Common.FunctionalTests/Utilities/SkipIISAttribute.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
     [AttributeUsage(AttributeTargets.Assembly | AttributeTargets.Class | AttributeTargets.Method)]
     public sealed class SkipIISAttribute : Attribute, ITestCondition
```

Comment:
We don't need this attribute. We need capability aware one. Like in:

`[RequireIISCapability(IISCapability.PoolEnvironmentVariables | IISCapability.WebSockets_]`

Reply:
Yeah this is significantly better.